### DISCUSSION TITLE
Where do you want to go if the coronavirus has converged?

### DISCUSSION THREAD

**Discussion by TakumiHiraide** [NAIST-SE/AttractingContributors#228] [General]
Please help me.
https://github.com/TakumiHiraide/Where-do-you-want-to-go-

**Comment by hernessteve** [comment]
I would love to go to Australia. I guess for that i need to call [speedy cash phone number](https://speedy-cash.pissedconsumer.com/customer-service.html) cause at the moment my income is pretty low and Australia is pretty far. And closed. But we are talking about the magical day when everything will be over, right?

**Comment by ellielo** [comment]
I would love to go backpacking to Europe. To stop in many different places and not worry about masks, vaccines etc. I am taking it very seriously with the quarantining now. I also have a super healthy lifestyle with all the fresh homemade foods and vitamins from [canadian pharmacy](https://www.canadapharmacy.com/) so hopefully i can survive the virus if it gets me one day.

**Comment by Ozankey** [comment]
I'm going to visit my friends in Europe.
